instrumentation/github.com/astaxie/beego/otelbeego/common.go

```diff
@@ -23,10 +23,6 @@ import (
 // ------------------------------------------ Constants
 
 const (
-	// packageName is the name of the this package, and is used as the default tracer
-	// and meter names.
-	packageName = "go.opentelemetry.io/contrib/instrumentation/github.com/astaxie/beego/otelbeego"
```

Comment:
`packageName` was not used anywhere (apart from tests), and with reference to https://github.com/open-telemetry/opentelemetry-go-contrib/pull/303#issuecomment-680990488 this is desired behavior, so I removed it.